.github/workflows/charts-lint-test-scan.yml

```diff
@@ -0,0 +1,18 @@
+name: Lint, test, and scan Helm charts
+on:
+  pull_request:
+    branches:
+      - master
+      - release/**
+  schedule:
+    - cron: "0 0 * * *"
+  workflow_dispatch:
```

Comment:
Wish someone could tell me why workflow_dispatch is there but empty. I know it's a boilerplate form, but I'd still like to know.

Reply:
When it is empty that means you can kick off a github action, and there are no inputs that are necessary to kick off the action.

Reply:
https://github.blog/changelog/2020-07-06-github-actions-manual-triggers-with-workflow_dispatch/